operator/v1/types_ingress.go

```diff
@@ -344,6 +344,52 @@ type IngressControllerSpec struct {
 	// +kubebuilder:default:="Immediate"
 	// +default="Immediate"
 	IdleConnectionTerminationPolicy IngressControllerConnectionTerminationPolicy `json:"idleConnectionTerminationPolicy,omitempty"`
+
+	// closedClientConnectionPolicy maps directly to HAProxy’s "abortonclose" option
+	// and controls how TLS handshakes and request processing are handled when the
+	// client closes the connection.
+	//
+	// Allowed values for this field are "Abort" and "Continue".
+	// The default value is "Continue".
+	//
+	// When set to "Abort", the router will avoid or stop performing the TLS handshake
+	// and will abort any HTTP request that has not yet been sent to the backend
+	// when the client closes the connection.
+	//
+	// When set to "Continue", the router will complete the TLS handshake and wait
+	// for the backend server's response regardless of the client having closed
+	// the connection.
```

Comment:
HAProxy will not send the HTTP request if the client closes the TCP connection while the TLS handshake with the client is in progress, right?
```suggestion
	// When set to "Continue", the router will complete the TLS handshake if
	// it is in progress or wait for the backend server's response if the
	// request has been sent, regardless of the client's having closed the
	// connection.
```